cfspopcon/formulas/metrics/greenwald_density.py

```diff
@@ -42,6 +30,20 @@ def calc_greenwald_density_limit(plasma_current: float, minor_radius: float) ->
         minor_radius: [m] :term:`glossary link<minor_radius>`
 
     Returns:
-        nG Greenwald density limit [n20]
+        :term:`greenwald_density_limit` [1e20 m^-3]
     """
     return plasma_current / (np.pi * minor_radius**2)
+
+
+@Algorithm.register_algorithm(return_keys=["average_electron_density"])
+def calc_average_electron_density_from_greenwald_fraction(greenwald_fraction: Unitfull, greenwald_density_limit: Unitfull) -> Unitfull:
+    """Calculate the average electron density corresponding to a given Greenwald fraction.
+
+    Args:
+        greenwald_fraction: :term:`glossary link<greenwald_fraction>`
+        greenwald_density_limit: :term:`glossary link<greenwald_density_limit>`
+
+    Returns:
+        :term:`average_electron_density` [1e20 m^-3]
```

Comment:
What do you think about the following? 

```suggestion
        :term:`average_electron_density` in same units as `greenwald_density_limit`
```